Jenkinsfile

```
@@ -33,7 +33,7 @@ pipeline {
     booleanParam(name: 'test_ci', defaultValue: false, description: 'Enable test')
     booleanParam(name: 'release_ci', defaultValue: true, description: 'Enable build the release packages')
     booleanParam(name: 'its_ci', defaultValue: true, description: 'Enable async ITs')
-    choice(name: 'build_type', choices: ['snapshot', 'staging'], description: 'Choose Snapshot or Staging Build type(Default: Snapshot)')
+    choice(name: 'build_type', choices: ['both', 'snapshot', 'staging'], description: 'Choose Snapshot or Staging Build type(Default: Both)')
```

I suggest a clearer name for "both", and also utilizing the actual default value.
```suggestion
    choice(name: 'build_type', choices: ['snapshot+staging', 'snapshot', 'staging'], description: 'Choose Snapshot or Staging Build type(Default: Both)', defaultValue: 'snapshot+staging')
```